### DISCUSSION TITLE
Possible to get the Wallet class from ethers from this lib?

### DISCUSSION THREAD

**Discussion by lsbyerley** [wevm/wagmi#144] [General]
This may be a somewhat dumb question but I'm wondering if it is possible to get the ethers Wallet class (https://docs.ethers.io/v5/api/signer/#Wallet) from wagmi..  or at least create one from the connected account? I've been trying to integrate a zkSync account for the user (https://zksync.io/api/sdk/js/accounts.html#wallet) and it seems i need the users private key to create a Wallet class from ethers. Any tips or help is appreciated

**Comment by brunoalano** [comment]
@lsbyerley Did you found some way to solve this?

**Comment by o-az** [comment]
You are already importing ethers when you use this library so why not use wallet class directly from ethers?
